src/main/webapp/index.xhtml

```diff
@@ -0,0 +1,213 @@
+    <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet"/>
+    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet"/>
+</h:head>
+
+<h:body>
+    <div class="header">
+        <div class="header-title header-text-2">Microprofile Starter <span
+                class="header-yellow">"Beta"</span></div>
+        <div class="header-title header-text-1 header-yellow">Generate MicroProfile Maven Project with Examples</div>
+        <a href="" class="feedback">
+            <span class="ui-button-text ui-c">Give Feedback</span>
+        </a>
+        <a href="https://microprofile.io" class="logo">
+            <img src="resources/img/MP-logo-w-tagline-1.png" alt="MicroProfile" height="50"/>
+        </a>
+    </div>
+
+    <div class="page-wrapper">
+        <h:form id="frm">
+
+            <div class="mainArea">
+
+                <div class="content">
+                    <p:messages showSummary="true" showDetail="false"/>
+                    <p:panelGrid columns="2" columnClasses="ui-grid-col-6 label-row, ui-grid-col-6 label-row" layout="grid"
+                                 styleClass="ui-panelgrid-blank ui-fluid">
+                        <p:outputLabel value="groupId" for="groupId"/>
+                        <p:outputLabel value="artifactId" for="artifactId"/>
+                    </p:panelGrid>
+
+                    <p:panelGrid columns="2" columnClasses="ui-grid-col-6, ui-grid-col-6" layout="grid"
+                                 styleClass="ui-panelgrid-blank ui-fluid">
+                        <p:inputText id="groupId" value="#{generatorDataBean.mavenData.groupId}" required="true"
+                                     title="Used as part of the package name so it must comply with the Java rules">
+                            <f:validator validatorId="packageNameValidator"/>
+                        </p:inputText>
+
+                        <p:inputText id="artifactId" value="#{generatorDataBean.mavenData.artifactId}" required="true"
+                                     title="Used as part of the package name so it must comply with the Java rules">
+                            <f:validator validatorId="packageNameValidator"/>
+                        </p:inputText></p:panelGrid>
+
+                    <p:panelGrid columns="2" columnClasses="ui-grid-col-6 label-row, ui-grid-col-6 label-row" layout="grid"
+                                 styleClass="ui-panelgrid-blank ui-fluid">
+                        <p:outputLabel value="MicroProfile Version" for="mpVersion"/>
+                        <p:outputLabel value="Java SE Version" for="javaSEVersion"/>
+                    </p:panelGrid>
+
+                    <p:panelGrid columns="2" columnClasses="ui-grid-col-6, ui-grid-col-6" layout="grid"
+                                 styleClass="ui-panelgrid-blank ui-fluid">
+                        <p:selectOneMenu id="mpVersion" value="#{generatorDataBean.mpVersion}"
```

Comment:
@rdebusscher Why haven't you pushed the latest version while at it? I mean the thing that I cherry-picked from you, e.g. ```generatorDataBean.engineData.mpVersion```.  Or do you expect that to arrive with #42 ?      Works for me, I'm just curious.

Reply:
The use of that engineData class should be in your PR as you mention.

And I have locally here now too many branches and things overlapping. So I wanted to push this Layout PR already (because already related to 5 issues) before I continue.

I will merge your PR in after this one is on master.

Reply:
:+1: